_grab_ui.py

```diff
@@ -383,155 +382,23 @@ def __call__(self, key: OptionName, value: Any) -> Any:
-    k('page up', 'page_up', 'move page up')
-    k('page down', 'page_down', 'move page down')
-    k('start of line', 'home', 'move first')
-    k('first non-whitespace', 'a', 'move first nonwhite')
-    k('last non-whitespace', 'end', 'move last nonwhite')
-    k('end of line', 'e', 'move last')
-    k('start of buffer', 'ctrl+home', 'move top')
-    k('end of buffer', 'ctrl+end', 'move bottom')
-    k('word left', 'ctrl+left', 'move word left')
-    k('word right', 'ctrl+right', 'move word right')
-    k('scroll up', 'ctrl+up', 'scroll up')
-    k('scroll down', 'ctrl+down', 'scroll down')
-    k('select left', 'shift+left', 'select stream left')
-    k('select right', 'shift+right', 'select stream right')
-    k('select up', 'shift+up', 'select stream up')
-    k('select down', 'shift+down', 'select stream down')
-    k('select page up', 'shift+page_up', 'select stream page up')
-    k('select page down', 'shift+page_down', 'select stream page down')
-    k('select to start of line', 'shift+home', 'select stream first')
-    k('select to first non-whitespace', 'A', 'select stream first nonwhite')
-    k('select to last non-whitespace', 'shift+end', 'select stream last nonwhite')
-    k('select to end of line', 'E', 'select stream last')
-    k('select to start of buffer', 'shift+ctrl+home', 'select stream top')
-    k('select to end of buffer', 'shift+ctrl+end', 'select stream bottom')
-    k('select word left', 'shift+ctrl+left', 'select stream word left')
-    k('select word right', 'shift+ctrl+right', 'select stream word right')
-    k('column select left', 'alt+left', 'select columnar left')
-    k('column select right', 'alt+right', 'select columnar right')
-    k('column select up', 'alt+up', 'select columnar up')
-    k('column select down', 'alt+down', 'select columnar down')
-    k('column select page up', 'alt+page_up', 'select columnar page up')
-    k('column select page down', 'alt+page_down', 'select columnar page down')
-    k('column select to start of line', 'alt+home', 'select columnar first')
-    k('column select to first non-whitespace', 'alt+A', 'select columnar first nonwhite')
-    k('column select to last non-whitespace', 'alt+end', 'select columnar last nonwhite')
-    k('column select to end of line', 'alt+E', 'select columnar last')
-    k('column select to start of buffer', 'alt+ctrl+home', 'select columnar top')
-    k('column select to end of buffer', 'alt+ctrl+end', 'select columnar bottom')
-    k('column select word left', 'alt+ctrl+left', 'select columnar word left')
-    k('column select word right', 'alt+ctrl+right', 'select columnar word right')
-
-    g('colors')
-    o('selection_foreground', '#FFFFFF', option_type=to_color)
-    o('selection_background', '#5294E2', option_type=to_color)
-
-    g('behavior')
-    o('select_by_word_characters',
-      json.loads(os.getenv('KITTY_COMMON_OPTS'))['select_by_word_characters'],
-      option_type=str)
-
-    type_map = TypeConvert({o.name: o.option_type
```

Comment:
If this class is no longer used, please delete its definition as well.

Reply:
Done in 1c758929df36d24bcaa2956e55613c18e723a107